packages/manager/apps/container/src/container/legacy/server-sidebar/SidebarVirtual.tsx

```diff
@@ -36,7 +36,7 @@ export default function ServerSidebarVirtual({
       } else {
         item.isOpen = !item.isOpen;
       }
-      if (item.href) {
+      if (item.href && !item.isExternal) {
```

Comment:
What happens for external link ?
Is there a specific click handler defined somewhere else ?

Reply:
Indeed for external links, the behavior is defined in ServerSidebarItemRenderer component